src/utils/check.your.answers.ts

```diff
@@ -153,11 +152,13 @@ const getBackLinkUrl = (isNoChangeJourney: boolean, hasAnyBosWithTrusteeNocs: bo
     return UPDATE_NO_CHANGE_REGISTRABLE_BENEFICIAL_OWNER_URL;
   } else {
     let backLinkUrl: string;
-    backLinkUrl = (isActiveFeature(FEATURE_FLAG_ENABLE_UPDATE_TRUSTS) && hasAnyBosWithTrusteeNocs) ? UPDATE_TRUSTS_ASSOCIATED_WITH_THE_OVERSEAS_ENTITY_URL : UPDATE_BENEFICIAL_OWNER_TYPE_URL;
+    backLinkUrl = (isActiveFeature(FEATURE_FLAG_ENABLE_UPDATE_TRUSTS) && hasAnyBosWithTrusteeNocs) ?
+      UPDATE_TRUSTS_ASSOCIATED_WITH_THE_OVERSEAS_ENTITY_URL :
+      UPDATE_BENEFICIAL_OWNER_TYPE_URL;
```

Comment:
line 155 - 157 is causing a code smell, it's my fault as I thought this section was still needed and advised it to be re-added. But looking at the flow of code, the backLinkUrl gets overwritten by the if(isRemove) / else blocks.
So lines 155-157 can be removed, that should fix the code smell

Reply:
Removed this.